docs/wiki/feature/arsenal.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: wiki
+title: ACE Arsenal
+description: Adds an interface that allow player to edit their loadouts.
+group: feature
+category: general
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 12
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Features
+
+ACE Arsenal has a pretty large number of improvements over BI Virtual Arsenal, here is a non exhaustive list:
+- The ability to select the magazine loaded into your primary / secondary / handgun without leaving ACE Arsenal.
+- Search bars for both the left and right panels.
+- New tabs to add weapon accessories to a container.
+- A clear container button to remove all items from a container.
+- Unique items support.`*`
+- Loadouts can be shared with other players.
+- Mission makers can save `Default loadouts` that will be available in their own tab independently from player saved loadouts.
```

Comment:
I wouldn't use inline code for marking special words

Reply:
indeed it may not be the best solution here's a comparison between inline code and simple quotes:
https://imgur.com/a/8DjWV
i'll switch them to quotes for now.